CHANGELOG.md

```diff
@@ -53,8 +53,9 @@ For more information about this file see also [Keep a Changelog](http://keepacha
   - Modularized input record, format record, and dbfiles record into shiny modules. This allows the app to be greatly simplified to two, single-page workflows. These functions can also be used "plug-and-play" style elsewhere in PEcAn shiny apps to load in data. 
   - Replaced modularized input, format and dbfiles records with static "Ingest Workflow" page. On this page, the user can select either importing from dataONE or Uploading from local files. If creating a new format is necessary, the user can click "Create New Format" and a dropdown menu will walk them through this process. 
   - Selected files now autofill name value in input record workflow
-  - Store inputs in the global environment
+  - Store inputs and formats in the global environment
   - "Test BETY" button allows users create a record in BETY with `dbfile.input.insert`
+  - Added separate `db.query` to create format record. 
```

Comment:
Would be more helpful if you named the new function in the log, rather than just referencing the functions called within your new function

Reply:
I will change this when I rewrite the function db.files.